### PR TITLE
Component binding is now case sensitive

### LINKED ISSUE #13457: [Preview 8] Component binding is now case-sensitive
More details: https://github.com/aspnet/AspNetCore/issues/9860

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #13457

*This is directed to the `release/3.0-preview8` branch.*

### CODE REVIEW

**Comment by guardrex** [comment]
@SteveSandersonMS @rynowak Looks like Ajay is out at the moment. Are either of u free for a quick look? It's a short few sentences and 2 minute review.

File: aspnetcore/blazor/components.md [hunk 1]
```diff
@@ -83,6 +85,8 @@ The following markup in *Index.razor* renders a `HeadingComponent` instance:
 
 [!code-cshtml[](common/samples/3.x/BlazorSample/Components/HeadingComponent.razor)]
 
+If a component contains an HTML element with an uppercase first letter that doesn't match a component name, a warning is emitted indicating that an element has an unexpected name&mdash;adding an `@using` statement for the component's namespace overrides the warning.
```
Comment (rynowak) [inline aspnetcore/blazor/components.md]: > adding an `@using` statement for the component's namespace overrides the warning.

This isn't really accurate. The warning points out that you attempted to bind a component, but that component wasn't found. Adding a using isn't a way to *suppress* the warning, it fixes the underlying problem (component was not found).

**Comment by guardrex** [comment]
@rynowak 

> Adding an `@using` statement for the component's namespace makes the component available, which removes the warning.

Better?

**Comment by rynowak** [comment]
Yes, thanks!

**Comment by guardrex** [comment]
Yes ... thanks. I'll add that now.

**Comment by guardrex** [comment]
ping @Rick-Anderson

**Pull request merged**